tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "include": ["src", "types"],
+  "compilerOptions": {
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "node",
+    "jsx": "preserve",
+    "baseUrl": "./",
+    /* paths - import rewriting/resolving */
+    "paths": {
+      // If you configured any Snowpack aliases, add them here.
+      // Add this line to get types for streaming imports (packageOptions.source="remote"):
+      //     "*": [".snowpack/types/*"]
+      // More info: https://www.snowpack.dev/guides/streaming-imports
+    },
+    /* noEmit - Snowpack builds (emits) files, not tsc. */
+    "noEmit": true,
+    /* Additional Options */
+    "strict": true,
+    "skipLibCheck": true,
```

Comment:
Why is this on `true`?

Reply:
It comes `true` by snowpack's bootstrap

I tried turning it to `false` just now, nothing seems to go wrong with building/testing

It seems that TypeScript currently recommends this setting: https://www.typescriptlang.org/tsconfig#skipLibCheck

Reply:
"_This can save time during compilation at the expense of type-system accuracy._"
Given that Snowpack is lightning fast already I would like to set this one to false.